azure-pipelines.yml

```diff
@@ -0,0 +1,52 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+variables:
+  FLUTTER_INSTALL_PATH: flutter
+  FLUTTER_TOOL_PATH: $(FLUTTER_INSTALL_PATH)/bin
+
+trigger:
```

Comment:
@boiledbuns Just to clarify, you want to run the CI only when there's an active PR for a branch and changes are made to the PR and when changes are made on the `master` right? Rather than running the CI on branches even though there's no PR for it?

Reply:
Hmmm good catch